Cargo.toml

```diff
@@ -40,7 +40,7 @@ tokio = { version = "1.35.1", default-features = false }
 tracing = "0.1.40"
 tracing-subscriber = "0.3.18"
 up-rust = "0.2.0"
-zenoh = { version = "=1.0.0-beta.3", features = ["unstable", "internal"] }
+zenoh = { version = "=1.0.0", features = ["unstable", "internal"] }
```

Comment:
why do you want to pin this to exactly `1.0.0`? Given that Zenoh now uses semantic versioning, it should also work with `1.0.14` or `1.7.5`, shouldn't it?

Reply:
Yes, I think it's fine to remove it now. The `=` was added due to the unstable changes in alpha / beta before.

Reply:
Updated